### PR TITLE
Add --min_as option to convolute

### PR DESCRIPTION
**Pull request by scarlehoff**
Although maybe the way I'm doing it is a bit... convoluted (canned laughter)

### CODE REVIEW

File: src/pineko/cli/convolute.py [hunk 1]
```diff
@@ -26,14 +27,18 @@
...
     """Convolute PineAPPL grid and EKO into an FK table.
 
     GRID_PATH and OP_PATH are the path to the respective elements to convolute, and
     FKTABLE is the path where to dump the output.
 
     MAX_AS and MAX_AL are used to specify the order in QCD and QED
-    couplings (i.e. the maximum power allowed for each correction).
+    couplings (i.e. the maximum power allowed for each correction)
+    e.g., max_as = 1, max_al = 0 would select LO QCD only
+    max_as = 3 instead would select LO, NLO, NNLO QCD
```
Comment (andreab1997) [inline src/pineko/cli/convolute.py:41]: Can you a comment also about `min_as`?

**Review by andreab1997 — CHANGES_REQUESTED** [review]
In general, I was not aware that this feature could be needed. Is there a specific application for which we would need to select only orders beyond, for example, NLO?

**Comment by scarlehoff** [comment]
Yes. If you want to create an fktable with only the NNLO contribution for instance. It is not usual (for instance when doing a theory you won't) but we (w/ @cschwan) are using it now to study the NNLO of DY as a function of q.

This is equivalent to doing in pineappl:

```
pineappl evolve ${grid} ${eko} ${fktable} NNPDF40_nnlo_pch_as_01180 --orders "a2as2"
```

**Comment by andreab1997** [comment]
> Yes. If you want to create an fktable with only the NNLO contribution for instance. It is not usual (for instance when doing a theory you won't) but we (w/ @cschwan) are using it now to study the NNLO of DY as a function of q.
> 
> This is equivalent to doing in pineappl:
> 
> ```
> pineappl evolve ${grid} ${eko} ${fktable} NNPDF40_nnlo_pch_as_01180 --orders "a2as2"
> ```
> 
> in pineko it would become
> 
> ```
> pineko convolute ${grid} ${eko} ${fktable} 3 1 --min_as 3
> ```

Yes, I guess that in `pineko` this is surely not the main feature so it makes sense to implement it as an optional argument. It is fine for me to merge this as soon as the docstrings are updated.

**Review by andreab1997 — APPROVED** [review]

**Pull request merged**